### DISCUSSION TITLE
Management of uploaded files

### DISCUSSION THREAD

**Discussion by lozdown** [sqlpage/SQLPage#577] [Q&A] [answered]
Hello,
I am using  sqlpage.persist_uploaded_file to upload files to sqlpage server location for database processing. Once processing is complete, for good server house keeping, the files should be discarded. What is a good way to use sqlpage to delete the uploaded files from the sqlpage server location?

**Comment by lovasoa** [comment]
Hello! 
  Can you tell us a little bit more about your use case,  how and when the files are processed? 

  To delete a file synchronously, you can use sqlpage.exec.

**Comment by lozdown** [comment, marked as answer]
My use case is very basic, I upload a csv file, import data to a postgresql database and discard the csv file. 
sqlpage.exec works fine. 
Thank you and thank you for creating sqlpage - it is a great tool
